versions.tf

```diff
@@ -6,7 +6,7 @@ terraform {
     }
     aws = {
       source = "hashicorp/aws"
-      version = "~> 2.0"
+      version = "~> 4.0"
```

Comment:
Was this originally 4.19.0?

Reply:
We made it 4.0, the ~> allows a looser dependency that allows any version >= 4.0 but less than 5